docs/dev-guide/golang/features.mdx

```diff
@@ -140,6 +140,7 @@ In this section you can find the following:
 - [How to Asynchronously complete an Activity](#asynchronous-activity-completion)
 - [How to register Namespaces](#namespaces)
 - [How to develop with Updates](#updates)
+- [How to use Start Delay](#start-delay)
```

Comment:
@flossypurse Do we need to update this as well to `Start Delay`? Not sure what the new format is.

Reply:
This page is going to be broken up anyways soon, so I wouldn't worry about this particular list atm.